app/services/shop.rb

```diff
@@ -0,0 +1,45 @@
+require "net/http"
+require "uri"
+
+class Shop
+  IMAGES_URL = "http://jsonplaceholder.typicode.com/photos"
+  TODOS_URL = "http://jsonplaceholder.typicode.com/todos"
+  IMAGES_COUNT = 5000
+
+  def self.buy(user, product)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `product`. If it's necessary, use `_` or `_product` as an argument name to indicate that it won't be used.